recipe/meta.yaml

```diff
@@ -9,36 +9,30 @@ package:
 
 source:
   fn: SuiteSparse-{{ version }}.tar.gz
-  url: https://github.com/jlblancoc/suitesparse-metis-for-windows/archive/{{ commit }}.tar.gz  # [win]
-  sha256: ff150a85b26e9c6a87a56770d5e26b08fd1208b35da8c28d37a436142a65e2b7  # [win]
+  url: https://github.com/jlblancoc/suitesparse-metis-for-windows/archive/v{{ bundle_version }}.tar.gz  # [win]
+  sha256: 1e1a901e62ac368c8c525bd43cf900e2e1133d2b20a58d5254561bb585af9a9c  # [win]
   patches:
     - gh_56.patch  # [win]
 
 build:
-  skip: true  # [win32 or unix]
-  number: 202
+  number: 0
   features:
     - blas_{{ variant }}
-  skip: True  # [win and vc<14]
+  skip: True  # [win and vc<14 or win32 or unix]
 
 requirements:
   build:
     - cmake
-    - ninja  # [win]
-    - {{ compiler('fortran') }}
+    - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
     - blas 1.1 {{ variant }}
     - openblas
-    - tbb
-    - metis
 
   run:
     - blas 1.1 {{ variant }}
     - openblas
```

Comment:
This needs to be changed to the blas scheme mentioned in the docs

Reply:
@isuruf Are you talking about this? https://conda-forge.readthedocs.io/en/latest/blas.html Is there any existing PR as the example how to properly add blas as dependency? I did read the doc, however I am still scratching my head how to do it right in `conda-build` recipe.